style.css

```diff
@@ -650,15 +650,156 @@ img {
 .finalframe {
   width: 327px;
   height: 72px;
-  flex: none;
   order: 1;
   flex-grow: 0;
 }
 
 .skillframe {
   width: 327px;
   height: 72px;
-  flex: none;
   order: 2;
   flex-grow: 0;
 }
+
+.contact-me {
+  background: url(./images/Contactme1.png);
+  margin-top: 70px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  width: 375px;
+  height: 750px;
+  text-align: center;
+  color: white;
+  margin-right: 50px;
+}
+
```

Comment:
[OPTIONAL]
- [ ] Good job attempting to fix the changes regarding the spacing around the contact form but there is one more to meet the required changes of the reviewer. I suggest that you remove the `margin-right: 50px` to get rid of the space.

Your Design | Figma Design
--|--
![image](https://user-images.githubusercontent.com/29071984/194275272-70fbefbe-276c-4f1e-a9ba-b8aed8447ebe.png) | <img src="https://user-images.githubusercontent.com/81727988/194024638-cb652a4d-42ca-4514-8791-c9a7aeec3f07.png" />